docker/config_template.yaml

```diff
@@ -48,6 +48,29 @@ persistence:
                 connectAttributes:
                     tx_isolation: 'READ-COMMITTED'
                 {{- end }}
+        {{- else if eq $db "postgres" }}
+        default:
+            sql:
+                pluginName: "postgres"
+                databaseName: {{ default .Env.DBNAME "cadence" }}
+                connectAddr: "{{ default .Env.POSTGRES_SEEDS "" }}:{{ default .Env.DB_PORT "5432" }}"
```

Comment:
Maybe this should be `SQL_*` to match the env vars in the SQL tool?

Reply:
I think this is fine. We use POSTGRES_SEEDS in https://github.com/uber/cadence/blob/master/environment/env.go#L62